app/views/projects.html

```diff
@@ -170,7 +170,12 @@ <h2 class="h1">
                 OpenShift helps you quickly develop, host, and scale applications.<br>
                 <span ng-if="canCreate">Create a project for your application.</span>
               </p>
-              <a ng-if="canCreate" href="create-project" class="btn btn-lg btn-primary">Create Project</a>
+              <div>
+                <button ng-click="createProject($event)" class="btn btn-md btn-primary">
```

Comment:
I don't think there is a `btn-md` (unless it's something Patternfly has added to Bootstrap).

https://getbootstrap.com/docs/3.3/css/#buttons-sizes

Reply:
Changed it back to btn-lg